src/styles/main.scss

```diff
@@ -1,3 +1,23 @@
-body {
+@import "./stars.scss";
+@import "./card.scss";
+@import "./_variables.scss";
+@import "./nav.scss";
+@import "./catalog.scss";
+@import "./header.scss";
+
+* {
+  box-sizing: border-box;
+}
+
+body,
+p {
   margin: 0;
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 12px;
+}
+
+html {
```

Comment:
It would be better to keep font settings consistent. You have declared the font family as 'Roboto', sans-serif in the body style, do the same in the html style.